src/testdrive/src/format/avro.rs

```diff
@@ -121,3 +121,12 @@ pub fn json_to_avro(json: &JsonValue, schema: SchemaNode) -> Result<AvroValue, S
         )),
     }
 }
+
+/// Computes the set difference between two slices of [`AvroValue`]s, i.e.,
+/// `lhs - rhs`.
+///
+/// Required because `AvroValue` does not implement `Hash`, `Eq`, or `Ord`, and
+/// so using a standard set type to perform the set difference is not possible.
+pub fn set_difference<'a>(lhs: &'a [AvroValue], rhs: &'a [AvroValue]) -> Vec<&'a AvroValue> {
```

Comment:
Did anything change with Avro types such that we can write it in this way, or were we always able to write it like this and just happened to have a less optimal version before?

Reply:
Nothing changed AFAICT!

Reply:
I think this version is slightly different in maybe important ways. The orig version tracks the multiplicities, so if there are two copies of a record in one and three in another, it will complain about one.